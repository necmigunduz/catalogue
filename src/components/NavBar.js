import '../assets/styles/css-lib/borders.css';
import '../assets/styles/css-lib/colors.css';
import '../assets/styles/css-lib/positioning.css';
import '../assets/styles/css-lib/fonts.css';
import { FaHome } from 'react-icons/fa';

const NavBar = () => (
  <nav className="bord-1 bg-5d flx jsc-between pad-20">
    <a href="#Search" className="color-white no-decor whattoeat courgette-font fs-30 fw-900 bord-10 pad-20">WhatToEat</a>
    <span className="fs-40 courgette-cursive color-b1a pad-20">Cook with what you have in your fridge!</span>
    <a href="/" className="color-white no-decor home rad fs-30 pad-20 m-top-20">
      <FaHome />
    </a>
  </nav>
);

export default NavBar;
