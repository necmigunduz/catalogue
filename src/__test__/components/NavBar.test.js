import React from 'react';
import renderer from 'react-test-renderer';
import NavBar from '../../components/NavBar';

describe('Nav', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<NavBar />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
