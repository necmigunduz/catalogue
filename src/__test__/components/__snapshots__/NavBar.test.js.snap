// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders correctly 1`] = `
<nav
  className="bord-1 bg-5d"
>
  <h1>
    <a
      href="#Search"
    >
      WhatToEat
    </a>
  </h1>
  <a
    href="/"
  >
    Home
  </a>
</nav>
`;
